From 7859cd2070ac21b2a517011ba987f2394d004b1b Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Mon, 5 Mar 2012 19:57:26 +0530
Subject: [PATCH 053/182] ARM: OMAP3+: am33xx_hwmod: Correct the usb
 clkdm_name

Correct the usb clock domain name to "l3s_clkdm".

The USB peripheral doesn't have any seperate clock domain of its
own, it falls under "l3s_clkdm".

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/omap_hwmod_33xx_data.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
index 619512c..1c8f252 100644
--- a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
@@ -2926,7 +2926,7 @@ static struct omap_hwmod_ocp_if *am33xx_usbss_slaves[] = {
 static struct omap_hwmod am33xx_usbss_hwmod = {
 	.name		= "usb_otg_hs",
 	.class		= &am33xx_usbotg_class,
-	.clkdm_name	= "wkup_usb_clkdm",
+	.clkdm_name	= "l3s_clkdm",
 	.mpu_irqs	= am33xx_usbss_mpu_irqs,
 	.main_clk	= "usbotg_fck",
 	.flags		= HWMOD_CONTROL_OPT_CLKS_IN_RESET,
-- 
1.7.9.1

